.github/workflows/integration-test.yml

```diff
@@ -6,7 +6,7 @@ env:
   TEST_BROWSER_HEADLESS: 1
   CI: 1
   PLUGIN_NAME: opensearch-security
-  OPENSEARCH_INITIAL_ADMIN_PASSWORD: admin
+  OPENSEARCH_INITIAL_ADMIN_PASSWORD: myStrongPassword123!
```

Comment:
we should generate a random password here instead of a hardcoded value. wdyt?

Reply:
I think thats overkill for testing. This value needs to match the one [here](https://github.com/opensearch-project/security-dashboards-plugin/blob/main/cypress.config.js#L33).

Reply:
I think the main concern we should address is that it should be able to be a randomly generated string here and still work, which is what this PR is doing. I think it would be overkill for testing. @cwperks that is actually used for cypress tests - this is just running the backend server tests, so this value could be anything.